packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -59,6 +60,22 @@ export const flushFilePath = (filePath: string): string => {
   return nativePath;
 };
 
+export const isSymbolicLink = (path: string) => {
+  try {
```

Comment:
@mingxuanzhangsfdx this function needs to be optimized. It is visited quite a bit in our code.